packages/vats/src/lib-board.js

```diff
@@ -32,40 +31,50 @@ const calcCrc = num => {
 /**
  * Create a board to post things on.
  *
- * @param {number} [seed=0]
+ * @param {number | bigint} [firstSequence=0]
+ * @param {string} [prefix]
  * @returns {Board}
  */
-function makeBoard(seed = 0) {
+function makeBoard(firstSequence = 0, prefix = 'board0') {
```

Comment:
making this parameterized implies that we're not guaranteeing all board ids begin the same way. That implies software can't look at a board ID and know that it's a board ID.

#2014 talks about "distinguishing in the wild" which sounds like it's mostly for human consumption and allowing new prefixes or synonyms could be socialized.

Is it explicitly not an objective that board IDs can be detected as such by software? (i.e.. matching an invariant pattern) That seems worth including to me, but I'm new to this area.

Reply:
You're right that it's for human consumption: UI software should *not* require a particular boardID structure (except for 'string'), though a conventional suggestion is fine for assisting humans.

The parameters allow the creation of a board with different behaviour.  But the `home.board` on Agoric will have conventional behaviour.  This is the same motivation as not requiring the use of `RUN` in our contracts since they could be repurposed.  But typically on the Agoric chain, it will be `RUN`.